### PR TITLE
Enable rust cryptography on develop.element.io

### PR DESCRIPTION
**Pull request by richvdh**
Once this lands, *existing* sessions will continue to use legacy crypto, but *new* logins will use rust crypto.

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by richvdh** [comment]
We'll follow up with a similar change for Element Desktop Nightly, but it's slightly trickier to get urgent fixes into users' hands for nightly, so will give it a few days to settle.

File: element.io/develop/config.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
     },
     "privacy_policy_url": "https://element.io/cookie-policy",
     "features": {
-        "feature_video_rooms": true
+        "feature_video_rooms": true,
+        "feature_rust_crypto": true,
```
Comment (richvdh) [inline element.io/develop/config.json:52]:
```suggestion
        "feature_rust_crypto": true
```

**Review by andybalaam — APPROVED** [review]
All fingers crossed.

**Comment by richvdh** [comment]
A couple more things have come up which we want to investigate first:
 * https://github.com/vector-im/element-web/issues/26301
 * https://github.com/vector-im/element-web/issues/26293

**Comment by richvdh** [comment]
Even more blockers have been identified. Now tracking these with a label: https://github.com/vector-im/element-web/issues?q=is%3Aissue+is%3Aopen+label%3AZ-Element-R-Blocker

**Pull request merged**